apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/parser/graphql/ast/ast2introspection.kt

```diff
@@ -0,0 +1,250 @@
+import com.apollographql.apollo.compiler.parser.error.ParseException
```

Comment:
Should we call this file `IntrospectionSchemaBuilder.kt`

Reply:
I didn't pay super attention to the names of the `internal`/`private` types but that definitely requires some love. A pattern that I see emerging is classes that don't serve any other purpose than holding a `schema` and `fragments`. I started naming these `XXXScope`, `InputValueValidationScope`, `MergeTypeExtensionsScope`, etc.. No strong feeling there so I'm happy to change to `XXXBuilder` or something else.